src/SeleniumLibrary/__init__.py

```diff
@@ -385,11 +385,22 @@ def failure_occurred(self):
             self._running_on_failure_keyword = False
 
     @property
+    def driver(self):
+        """Current active driver
+
+        :rtype: selenium.webdriver.remote.webdriver.WebDriver
+        :raises RuntimeError if driver is not created
+        """
+        if not self._drivers.current:
+            raise RuntimeError('No driver is open')
```

Comment:
Strange error message. Perhaps this should still be `No browser is open.`? Also, should we use a dedicated exception?

Reply:
Yes, browser is better.

It's strange exception, but that is how the original code worked. Dedicated exception would be better, would `NoBrowser` be good?

Reply:
Perhaps NoOpenBrowser or NoBrowserOpen?